src/CoreArbiterServerMain.cc

```diff
@@ -118,6 +118,7 @@ int main(int argc, const char** argv) {
             printf(" %d", coresUsed[i]);
         putchar('\n');
     }
+    fflush(stdout);
```

Comment:
Why is this needed? Let's discuss. Same with with line 67 below.

Reply:
I think we need to standardize the logging: write to an error stream similar to Arachne rather than print to stdout.